nss-day-cohort-47/reactive-nutshell-palm-beach-pilots
# The final crud for the article component and helper function for sorting arrays --CD

# Description
This pull request includes all the crud integration for the articles component. User can now add, update and delete their article posts and see the posts of their friends in the style requested by customer. Also included is a helper function that can be used to sort arrays by specific key value in ascending or descending order.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
When the user logs in and select articles they should see their posts as well as posts of their friends. The user should now be able to delete add or edit any of their posts. Upon any of these actions the article list should be renewed to reflect the user's change.
# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
